### DISCUSSION TITLE
Copilot not appearing in VSC 1.74.1 on OSX Monterey 12.6

### DISCUSSION THREAD

**Discussion by michaelmoreno** [community/community#41962] [Copilot Conversations]
### Select Topic Area

Bug

### Body

The Copilot icon is missing from the status bar, autocomplete suggestions are absent, and the only command associated with copilot in f1 is `Collect Diagnostics`
<img width="602" alt="image" src="https://user-images.githubusercontent.com/72640530/208228010-1272c36a-f96b-49ee-a83f-0b060ae0cfe6.png">
which produces this error when ran
<img width="255" alt="image" src="https://user-images.githubusercontent.com/72640530/208228030-9e18385b-9a16-4364-baa4-aff97a8bc022.png">

This started recently as it was working earlier today.

Have tried reinstalling vscode and copilot extension.

**Comment by horridharry** [comment]
There was a recent update about today which appears to be a corrupted or broken version of Github Copilot. You need to install an older version of Github Copilot by going into the extension page thens earching for 'Github Copilot' and clicking the arrow button next to 'Uninstall'. From there, click the tab which says 'Install Another Version' and install v1.63.7601 (x days ago).

You will be prompted to reload VS Code and Github Copilot should begin working again.

**Reply by HemantSachdeva** [reply]
Thanks for the temporary solution. It works.

**Comment by guzzur** [comment]
I needed to type with my fingers for an hour before I found this solution! Thank you so much

**Comment by NomadicDeveloper22** [comment]
Happening in VSCode on Windows 10 as well. Definitely seems to be the broken update, started happening immediately after

**Comment by xiangzhang1** [comment]
Happening in VSCode on Ubuntu 22.04 LTS too. No login prompt, no code suggestion, command 'github.copilot.collectDiagnostics' not found. v1.63.7601 works.
